src/components/user/user.repository.ts

```diff
@@ -407,6 +407,7 @@ export const userFilters = filter.define(() => UserFilters, {
   }),
   title: filter.propPartialVal(),
   roles: filter.stringListProp(),
+  status: filter.propVal(),
```

Comment:
Ordering of these filters actually declares the order of the generated cypher, so we want cheaper filters first. i.e don't do the hard work of full-text matching a name for all users, when we are then going to filter that down to the ~20 disabled ones.

I'd say this could go after `pinned`.